core/test/truffle-fixture.js

```diff
@@ -0,0 +1,36 @@
+const deployFinder = require("../migrations/2_deploy_finder.js");
```

Comment:
Just so I understand: this basically lets the buidler tests access the truffle migrations by running through them all?

Reply:
This is so it adapts truffle Migrations to become a buidler-truffle fixture.